Add ratings distribution plot to the app

mikeguron opened the pull request.

louiewang820 reviewed (commented):
runtime_movies <- separate_rows(movies, Genre, sep = ",")
runtime_movies$Genre <- trimws(runtime_movies$Genre)

ratings_movies <- separate_rows(movies, Genre, sep = ",")
ratings_movies$Genre <- trimws(ratings_movies$Genre)

 runtime_data <- wrangled_data(runtime_movies, input)
 ratings_data <- wrangled_data(ratings_movies, input)

Maybe using one of them is sufficient. We don't need to create a separate data for this

mikeguron commented:
Yes, I agree with that suggestion.  I have updated the code to remove those redundancies and renamed the variables so they are suitable for use across multiple plots.
